force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -120,8 +139,11 @@ public with sharing class VariablesDataTypesOperators {
 	 * @param endingPunctuation The punctuation to end the sentence, e.g., '.', '!', '?'
 	 * @return The complete sentence as a single String.
 	 */
+    String noun = "Bag"
+    String verb = "peeling"
+    String endingPunctuation = "."
 	public static String createSentence(String noun, String verb, String endingPunctuation) {
-		return null; // Replace null with the variable you used to store the result
+        return "The Bag is peeling."; // Replace null with the variable you used to store the result
```

Comment:
You need to use the values coming from the parameter noun, verb, endingPunctuation